src/js/clickable-component.js

```diff
@@ -32,6 +33,10 @@ class ClickableComponent extends Component {
    * @param  {string} [options.className]
    *         A class or space separated list of classes to add the component
    *
+   * @param  {number | boolean} [options.throttle]
+   *         A throttle will be applied to the clickHander if the number is >= 1 or the value is `true`
+   *         A number specifies the desired wait time in ms or a default wait of 50ms will be applied
```

Comment:
any details on how 50 was chosen as the default?

Reply:
Nothing empirical, just a guess on my part based on previous similar solutions I've worked on. However, this is a good point so I measured the gap on my fairly modern Windows PC, and got 1-2 ms. Sadlt FF doesn't have the same performance spoofing options as Chrome (which doen't receive double input)  but maybe I can get more data from Browserstack or similar.

Reply:
My bad -  I had hoped Browserstack might have options to set hardware limitations, but it does not.